ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -72,13 +152,42 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("SignalProducer.try") {
-			pending("should run the operation once per start()") {
+			it("should run the operation once per start()") {
+				var operationRunTimes = 0
+				let operation: () -> Result<String, NSError> = {
+					operationRunTimes++
+
+					return Result<String, NSError>.Success(Box("OperationValue"))
```

Comment:
Mmmm what's the syntax to specify the "missing type" that it can't infer?

```swift
// Argument for generic type E can't be inferred
success("successValue")
```

```swift
// Cannot explicitly specialize a generic function
success<String, NoError>("successValue")
```